bash-test-dummy/visualise.R

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env Rscript
```

Comment:
Any reason to use R over Python? It's not important, as this is more like a paper artifact: it's important that it exists in the open, but I don't many users. It's more out of curiosity.

Reply:
Just that, surprisingly, I know R better than Python when it comes to this kind of work :wink: